packages/create-contentful-app/package.json

```diff
@@ -25,7 +25,7 @@
     "create-contentful-app": "index.js"
   },
   "dependencies": {
-    "@contentful/create-contentful-app": "0.9.10"
+    "@contentful/create-contentful-app": "file:../contentful--create-contentful-app"
```

Comment:
Does this work with npm (as opposed to yarn)? You might get 404s

Reply:
Still testing. What I know is that dependabot doesn't bump the version of other monorepo packages.

Reply:
lerna is using `file:` within their own repo (https://github.com/lerna/lerna) and use npm